docs/aws/aws-how-to/kubernetes/deploy-managed-node-group.rst

```diff
@@ -0,0 +1,187 @@
+
+..  code-block:: bash
+
+    aws eks update-kubeconfig --region us-east-1 --name My-Ubuntu-Cluster --profile eks
+
+Now you can monitor your cluster using ``kubectl`` as follows:
+
+..  code-block:: bash  
+
+    kubectl get nodes -o wide
+
+Wait for the node groups to be up and running. The next step is to test how to trigger the upgrading of the nodes.
+
+Part 2: Triggering automatic node replacement in an existing managed nodes group
+--------------------------------------------------------------------------------
+
+If your cluster is already running with managed node groups and you need to update the base AMI automatically, you’ll have to create a new version of the launch template and update the node group to use it. This will trigger the “cordon and drain” node replacement strategy automatically.
+
+
+Create a new launch template version
+""""""""""""""""""""""""""""""""""""
+
+To create a new version of the launch template, you need to use your existing launch template ID (created in the previous section), and update only the ``ImageId`` field with the new AMI ID. The CLI command to do this is:
+
+..  code-block:: bash
+
+    aws ec2 create-launch-template-version \
+        --launch-template-id lt-0123456789abcdef0 \
+        --version-description "Ubuntu Pro EKS image v2" \
+        --source-version 1 \
+        --launch-template-data '{
+            "ImageId":"ami-NEWAMI123456"
+        }'
+
+In the command, source-version is set to '1' since we are updating the originally created launch template. The new version created will have its version automatically set to '2'.
+
+Update the managed node group to use the new launch template version
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+To update the node group to use the new launch template version, run:
+
+..  code-block:: bash
+
+    eksctl upgrade nodegroup \
+        --name=my-managed-ng \
+        --cluster=My-Ubuntu-Cluster \
+        --region=us-east-1 \
+        --launch-template-version 2
+
+This triggers **automatic node replacement**, draining old nodes and launching new ones with the updated image.
+
+Future steps
+------------
+
+We have covered how using Launch Templates you can build your own managed nodes with Ubuntu, Ubuntu Pro and any customization you would need.
+
+Now that you understand the workflow, the next step is automation:
+
+- Use AWS EC2 Image Builder to generate new AMIs on a schedule.
+- Trigger EventBridge events that invoke a Lambda function to:
+
+  - Update the launch template with the new AMI.
+  - Point the managed node group to the new template version.
+
+This keeps your cluster always up-to-date with minimal manual effort.
+
+
+.. _`eksctl`: https://docs.aws.amazon.com/eks/latest/eksctl/installation.html
+.. _`AWS CLI`: https://documentation.ubuntu.com/aws/aws-how-to/instances/find-ubuntu-images/
```

Comment:
i think we can do relative links in tree. @k-dimple is it better to do that in case our urls change?